middleware/validation.js

```diff
@@ -84,6 +85,38 @@ export const validateLogin = [
     .withMessage('Please provide a valid password.'),
 ];
 
+export const validateUser = async (req, res, next) => {
```

Comment:
@Dubby20 this variable name gives the impression that you're just verifying a user, meanwhile, we're verifying that the user is the author of the article. Let's use a more descriptive method name.

Reply:
Alright. I will change the name to be more descriptive.